### PR TITLE
Feature player attacking sound

### PR DESCRIPTION
**Pull request by beerhond**
Sorry, forgot to merge this in the last sprint!

### CODE REVIEW

File: Confus/Confus/Audio/PlayerAudioEmitter.cpp [hunk 1]
```diff
@@ -45,6 +107,19 @@ namespace Confus
             m_AudioSourceFootstep1 = new OpenALSource("Footstep1_Concrete.wav");
             m_AudioSourceFootstep2 = new OpenALSource("Footstep2_Concrete.wav");
             m_AudioSourceFootstep3 = new OpenALSource("Footstep3_Concrete.wav");
+
+            m_AudioSourceGrunt1 = new OpenALSource("Grunt1.wav");
+            m_AudioSourceGrunt2 = new OpenALSource("Grunt2.wav");
+            m_AudioSourceGrunt3 = new OpenALSource("GruntHeavy.wav");
```
Comment (Athosvk) [inline Confus/Confus/Audio/PlayerAudioEmitter.cpp]: This one is not deleted in the destructor

File: Confus/Confus/Audio/PlayerAudioEmitter.cpp [hunk 2]
```diff
@@ -14,6 +14,12 @@ namespace Confus
             delete(m_AudioSourceFootstep1);
             delete(m_AudioSourceFootstep2);
             delete(m_AudioSourceFootstep3);
+            delete(m_AudioSourceGrunt1);
+            delete(m_AudioSourceGrunt2);
+            delete(m_AudioSourceSwordSwosh1);
+            delete(m_AudioSourceSwordSwosh2);
+            delete(m_AudioSourceSwordSwosh3);
+            delete(m_AudioSourceSwordSwosh4);
             delete(m_AttachedPlayer);
```
Comment (Athosvk) [inline Confus/Confus/Audio/PlayerAudioEmitter.cpp]: I thought this was/would be fixed; deleting memory that isn't allocated by yourself

File: Confus/Confus/Audio/PlayerAudioEmitter.h [hunk 3]
```diff
@@ -1,6 +1,7 @@
 #pragma once
 #include <irrlicht/irrlicht.h>
 #include <memory>
+#include <time.h>
```
Comment (Athosvk) [inline Confus/Confus/Audio/PlayerAudioEmitter.h]: This is only used in the cpp file. You might want to move the include

File: Confus/Confus/Audio/PlayerAudioEmitter.h [hunk 4]
```diff
@@ -17,6 +18,16 @@ namespace Confus
...
             OpenALSource* m_AudioSourceFootstep3;
+
+            OpenALSource* m_AudioSourceGrunt1;
+            OpenALSource* m_AudioSourceGrunt2;
+            OpenALSource* m_AudioSourceGrunt3;
+
+            OpenALSource* m_AudioSourceSwordSwosh1;
+            OpenALSource* m_AudioSourceSwordSwosh2;
+            OpenALSource* m_AudioSourceSwordSwosh3;
+            OpenALSource* m_AudioSourceSwordSwosh4;
```
Comment (Athosvk) [inline Confus/Confus/Audio/PlayerAudioEmitter.h]: Why are these actually not just arrays?

File: Confus/Confus/Audio/PlayerAudioEmitter.cpp (hunk 2 above)
Comment (beerhond) [inline Confus/Confus/Audio/PlayerAudioEmitter.cpp]: Fixed it now.

File: Confus/Confus/Audio/PlayerAudioEmitter.cpp (hunk 1 above)
Comment (beerhond) [inline Confus/Confus/Audio/PlayerAudioEmitter.cpp]: Fixed

File: Confus/Confus/Audio/PlayerAudioEmitter.h (hunk 3 above)
Comment (beerhond) [inline Confus/Confus/Audio/PlayerAudioEmitter.h]: Ok

File: Confus/Confus/Audio/PlayerAudioEmitter.h (hunk 4 above)
Comment (beerhond) [inline Confus/Confus/Audio/PlayerAudioEmitter.h]: True that.

**Pull request merged**